src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateGenerator.cs

```diff
@@ -40,7 +40,6 @@ public class CertificateGenerator
         // Give the cert a grace period in case there's a time skew between machines
         private readonly TimeSpan _gracePeriod = TimeSpan.FromHours(1);
 
-        private const string _authorityCanonicalName = "DO_NOT_TRUST_WcfBridgeRootCA";
```

Comment:
I don't understand the recommendation to make this settable.  Isn't it a const?  In fact, I don't understand the null test below against a const.  Maybe before refactoring, @StephenBonikowsky could you state in these PR comments what the test needs to do with respect to CertificateGenerator?  I thought you only needed to read this value.